packages/web-frontend/src/containers/MeasureLegend/MultiLegend.js

```diff
@@ -55,7 +55,9 @@ const getLegend = measureType => {
 };
 
 const MultiLegend = React.memo(({ measureOptions, isMeasureLoading }) => {
-  const displayedLegends = measureOptions.filter(({ type }) => type !== MEASURE_TYPE_RADIUS);
+  const displayedLegends = measureOptions.filter(
+    ({ type, hideFromLegend }) => type !== MEASURE_TYPE_RADIUS && hideFromLegend !== true,
+  );
```

Comment:
Quite surprised we didn't have this previously, as we attach this key to the measureOption by default...